govspeak.gemspec

```diff
@@ -29,7 +29,7 @@ library for use in the UK Government Single Domain project'
   s.executables   = s.files.grep(%r{^bin/}) { |f| File.basename(f) }
   s.require_paths = %w[lib]
 
-  s.add_dependency "actionview", ">= 5.0", "< 7"
+  s.add_dependency "actionview", ">= 5.0"
```

Comment:
We may as well bump this to `>= 6` since Rails 5 doesn't run on Ruby 2.7. It does seem reasonable to be optimistic with the versioning here.